pkgs/cronet_http/README_EMBEDDED.md

```diff
@@ -0,0 +1,23 @@
+An Android Flutter plugin that provides access to the
+[Cronet](https://developer.android.com/guide/topics/connectivity/cronet/reference/org/chromium/net/package-summary)
+HTTP client.
+
+Cronet embedded is available as a standalone implementation
+which will brought additional 8MB for apps approximately.
+
+## Status: Experimental
```

Comment:
I'd remove this line and everything after it. It will be annoying to keep this file in sync with README.md

Reply:
I think we can leave at least a *more details* link like:

> See more details about the cronet_http at: https://pub.dev/packages/cronet_http.